Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
And I am a little worried that this 'ruby api' for google is in 'maintenance mode' (according to the README.md: https://github.com/google/google-api-ruby-client#library-maintenance) and that we should be using the youtube api directly, but if it works I guess it is OK.